src/main/webapp/app/admin/organization-management/organization-management.service.ts

```diff
@@ -0,0 +1,103 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpResponse } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { SERVER_API_URL } from 'app/app.constants';
+import { Organization } from 'app/entities/organization.model';
+
+@Injectable({ providedIn: 'root' })
+export class OrganizationManagementService {
+    public resourceUrl = SERVER_API_URL + 'api/organizations';
+
+    constructor(private http: HttpClient) {}
+
+    /**
+     * Send GET request to retrieve all organizations
+     */
+    getOrganizations(): Observable<any> {
+        return this.http.get(this.resourceUrl + '/all');
+    }
+
+    /**
+     * Send GET request to retrieve all organizations with their number of
+     * users and courses
+     */
+    getNumberOfUsersAndCoursesOfOrganizations(): Observable<any> {
```

Comment:
Since you know what the service returns, you can change `any` to something more specific like `number` (same for the functions below)

Reply:
Done with https://github.com/ls1intum/Artemis/pull/2748/commits/ab2264066656190b61efae207cf9a1382a0104d4 but could not come up with a type for `getNumberOfUsersAndCoursesOfOrganizations()` as response from the backend has form `Map<Long, Map<String, Long>>`